jeremy2918/tsseract
# Added manifest.json in the public next dir

# [Manifest](https://trello.com/c/KXVdjL0Z/54-i-want-to-set-the-right-address-for-the-apps-favicon-and-manifest)


This PR adds the right manifest file and directory on the document head for NextJS. Since non-TypeScript files are not compiled into the dist folder, I have added a npm script to add those files to `dist/client/public`.
## Testing Steps

- Pull and checkout to this branch.
- `npm install`
- `npm run tsc`
- `npm test`
- `npm run watch`
- At this time, you should be seeing no manifest or favicon in the app.
- Run `npm run copy-files:linux` and reload the page. Now the app should have a favicon and loaded the fill manifest with it's icons included.



## UI Changes
![Selection_038](https://user-images.githubusercontent.com/48720232/98036923-3648da00-1df1-11eb-94cc-fa9b146c1404.png)
